tests/modules/apigee_edge_entity_class/apigee_edge_entity_class.info.yml

```diff
@@ -0,0 +1,9 @@
+name: 'Apigee Edge: Entity class override test'
```

Comment:
Merge this with the Apigee Edge testing module and run all tests with the overridden classes and with that ensure that all functionalities are working properly even if someone overrides the entity classes.